app/src/main/java/com/hedvig/app/feature/insurance/ui/CrossSell.kt

```diff
@@ -143,10 +164,11 @@ private val previewData = InsuranceModel.CrossSell(
 )
 @Composable
 fun CrossSellPreview() {
+    val context = LocalContext.current
     HedvigTheme {
         CrossSell(
             data = previewData,
-            onCtaClick = {}
+            onCtaClick = { context.makeToast("Doing stuff") }
```

Comment:
Right, this was left from testing, should prob remove, which will remove the makeToast extension fun import too